pkg/tsdb/influxdb/flux/executor.go

```diff
@@ -33,7 +33,6 @@ func executeQuery(ctx context.Context, query QueryModel, runner queryRunner, max
 		dr.Frames = append(dr.Frames, metaFrame)
 		return
 	}
-	defer tables.Close()
```

Comment:
it doesn't hard to keep this, right?

Reply:
@papagian Per se, but I found out in an issue thread in the Go client repo that it's actually called internally, so clients aren't supposed to call it after all. See this [comment](https://github.com/influxdata/influxdb-client-go/issues/183#issuecomment-674053677) and the [code](https://github.com/influxdata/influxdb-client-go/pull/184/files/7d3656ca553768bc7ee7d4a4b80993f4b6222271#diff-8b3ced92a99034e241e8d5c5818effa7R228-R239) in question.

Reply:
I see